t.py

```diff
@@ -638,31 +668,32 @@ def permutate_with_flags(inputs: List[List[int]], flags_to_permutate: List[int])
         return [Input(i, f) for i in inputs for f in permut]
 
     @staticmethod
-    def generate_inputs(dynamic_operands: List[Operand]) -> List[List[int]]:
+    def generate_inputs(dynamic_operands: list[Operand]) -> list[list[int]]:
         dynamic_operands = len(dynamic_operands)
 
         if dynamic_operands == 0:
             return [[]]
         elif dynamic_operands == 1:
+            # TODO: changed to [i] but was i
```

Comment:
I had a type error in here, so I changed it from `[i for i in range(0, 1024)]` to `[[i] for i in range(0, 1024)]` but I'm not sure if that breaks the code or the old version was correct.

Reply:
Good spot! I think your version is correct, the previous one may have failed when generating the assembly file without me noticing (currently all exceptions there are ignored, as some invalid instructions will be generated and must be ignored, but it's not a great solution).

Reply:
Actually it probably makes sense to only ignore one specific error type of assembling (`CalledProcessError`) in `learn_single_flags` and log all the others, making it instantly noticeable when a lot of things go wrong